pepper-apis/src/main/resources/db-changes/schema/DDP-3709-file-upload.xml

```diff
@@ -23,16 +23,16 @@
             <column name="file_upload_guid" type="varchar(20)">
                 <constraints nullable="false" unique="true" uniqueConstraintName="file_upload_guid_uk"/>
             </column>
-            <column name="blob_name" type="varchar(300)">
+            <column name="blob_name" type="varchar(1024)">
                 <constraints nullable="false"/>
             </column>
-            <column name="mime_type" type="varchar(50)">
+            <column name="mime_type" type="varchar(255)">
```

Comment:
While digging around (see [here][1] and [here][2]), I found that mime type strings can be quite long, so bumping this up.

[1]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
[2]: https://stackoverflow.com/questions/643690/maximum-mimetype-length-when-storing-type-in-db